include/core/parameters.h

```diff
@@ -856,6 +881,20 @@ namespace Parameters
       temperature
     } variable;
 
+    // WIP Map containing the refinement variables
+    std::map<Variable, MultipleAdaptationParameters> variables;
+    // declaration for parsing variables
+    Variable                     vars;
+    MultipleAdaptationParameters var_adaptation_param;
+    // end WIP
+
+    // WIP remove when multivariable is implemented
+    // Coarsening fraction
+    double coarsening_fraction;
+    // Refinement fraction
+    double refinement_fraction;
+    // end WIP
```

Comment:
This could already be removed by using the values of the struct in the first element of the map.